<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loggerForm</class>
 <widget class="QWidget" name="loggerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>445</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QWidget" name="windowContainer" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="onOffButton">
          <property name="text">
           <string>Ligar/Desligar Bluetooth</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="findButton">
          <property name="text">
           <string>Encontrar dispositivos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="unpairButton">
          <property name="text">
           <string>Desemparelhar dispositvo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pairButton">
          <property name="text">
           <string>Emparelhar dispositivo</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QPlainTextEdit" name="logWindow">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
